specs-go/config.go

```diff
@@ -627,6 +627,19 @@ type WindowsCPUResources struct {
 	// cycles per 10,000 cycles. Set processor `maximum` to a percentage times
 	// 100.
 	Maximum *uint16 `json:"maximum,omitempty"`
+	// Set of CPUs to affinitize for this container.
+	Affinity []WindowsCPUGroupAffinity `json:"affinity,omitempty"`
+	// Specifies preferred set of numa node numbers to affinitize for this container.
+	AffinityPreferredNumaNodes []uint32 `json:"affinityPreferredNumaNodes,omitempty"`
```

Comment:
Is this supposed to be here still?

Reply:
Not sure why the PR is not updating :( was able to confirm from my local branch that these changes were infact removed. Checking what's happening